packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -106,7 +113,8 @@ export function updateMongoDBServerInfo() {
         'server-info',
         '--',
         '--connectionString',
-        `mongodb://127.0.0.1:${String(MONGODB_TEST_SERVER_PORT)}`,
+        process.env.TEST_ATLAS_CLOUD_EXTERNAL_CONNECTION_STRING_1 ||
+          `mongodb://127.0.0.1:${String(MONGODB_TEST_SERVER_PORT)}`,
```

Comment:
```suggestion
        DEFAULT_CONNECTION_STRING_1
```

Reply:
DEFAULT_CONNECTION_STRING_1 has the /test suffix in its definition, but the one here doesn't. 

I think that's supposed to refer to the auth DB that is used? if you think it won't make a difference then I can change it to what you suggested.